src/scenes/Partners/Detail/Tabs/Projects/PartnerDetailProjectsTable.tsx

```diff
@@ -0,0 +1,135 @@
+import { Shield as ShieldIcon } from '@mui/icons-material';
+import { useTheme } from '@mui/material/styles';
+import { DataGrid, DataGridProps, GridRowParams } from '@mui/x-data-grid';
+import { useNavigate } from 'react-router-dom';
+import { ProjectStatusLabels } from '~/api/schema/enumLists';
+import { PartnerDetailsFragment } from '../../PartnerDetail.graphql';
+import { PartnerDetailProjectsTableListItemFragment } from './PartnerDetailProjectsTable.graphql';
+
+export interface PartnerDetailProjectsTableProps {
+  projects: readonly PartnerDetailProjectsTableListItemFragment[];
+  GridProps?: Omit<DataGridProps<RowData>, 'columns' | 'isCellEditable'>;
+}
+
+export interface RowData {
+  id: string;
+  label: string;
+  data: PartnerDetailsFragment[];
+}
+
+export const PartnerDetailProjectsTable = ({
+  projects,
+}: PartnerDetailProjectsTableProps) => {
+  const navigate = useNavigate();
+  const theme = useTheme();
+
+  const handleRowClick = (params: GridRowParams) => {
+    navigate(`/projects/${params.row.id}`);
+  };
+
+  return (
+    <DataGrid<PartnerDetailProjectsTableListItemFragment>
+      density="compact"
+      initialState={{
+        sorting: {
+          sortModel: [{ field: 'name', sort: 'desc' }],
+        },
+      }}
+      disableColumnMenu
+      disableSelectionOnClick
+      autoHeight
+      hideFooter={true}
+      showCellRightBorder={false}
+      rowSpacingType="margin"
+      sx={{
+        border: 'none',
+        '& .MuiDataGrid-columnSeparator': {
+          display: 'none',
+        },
+        '& .MuiDataGrid-columnHeaders': {
+          borderBottom: 'none',
+        },
+        '& .MuiDataGrid-columnHeaderTitle': {
+          fontWeight: 'bold',
+        },
+        '& .MuiDataGrid-cell': {
+          borderBottom: 'none',
+        },
+        '& .MuiDataGrid-row:hover': { cursor: 'pointer' },
+        '& .MuiDataGrid-cell, & .MuiDataGrid-columnHeader': {
+          '&:focus, &:focus-within': { outline: 'none' },
+        },
+        '& .MuiDataGrid-columnHeader:nth-last-of-type(-n+2) .MuiDataGrid-columnSeparator--sideRight':
+          {
+            display: 'none',
+          },
+      }}
+      rows={projects}
+      onRowClick={handleRowClick}
+      columns={[
+        {
+          headerName: 'Project Name',
+          field: 'name',
+          flex: 1,
+          renderCell: (params) => {
+            const nameObj = params.row.name;
+            return (
+              <span style={{ color: theme.palette.primary.main }}>
+                {nameObj.value ?? ''}
+              </span>
+            );
+          },
```

Comment:
Using a `valueGetter` is a bit more idiomatic. Also a Box simplifies the color picking.
```suggestion
          valueGetter: ({ value }) => value.value,
          renderCell: ({ value }) => (
            <Box component="span" color="primary.main">
              {value}
            </Box>
          ),
```